src/ray/rpc/grpc_server.cc

```diff
@@ -58,8 +58,16 @@ void GrpcServer::Run() {
   // Build and start server.
   server_ = builder.BuildAndStart();
 
-  RAY_CHECK(server_) << "Failed to start the grpc server. The specified port is "
-                     << specified_port;
+  RAY_CHECK(server_)
+      << "Failed to start the grpc server. The specified port is " << specified_port
+      << ". Grpc servers are used for ray's core components such as raylet, gcs "
+         "sever, or worker processes. If the server startup error message is Address "
```

Comment:
(we don't need to dive into too much detail here imo)
```suggestion
      << ". This means that Ray's core components will not be able to function correctly." If the server startup error message is Address "
```